client/src/App.tsx

```diff
@@ -1,26 +1,31 @@
 import "./App.scss";
 import PlayerScore from "./components/PlayerScore/PlayerScore";
-import { DEFAULT_ROUNDS } from "./constants/constants";
+import { DEFAULT_ROUNDS, DEFAULT_PLAYERS } from "./constants/constants";
 import { Header } from "./components/header/Header";
+import { initialiseGame } from "./service/game";
 function App() {
+  const game = initialiseGame(DEFAULT_PLAYERS, DEFAULT_ROUNDS);
+  const player1 = game.players[0];
+  const player2 = game.players[1];
+
   return (
     <>
       <PlayerScore
```

Comment:
@beccarothwell I was thinking we could pass to the PlayerScore the player as an object and then get the name, id, score in the component? What do you think?

Reply:
I considered this at the time, but it was before we defined types. That makes sense though, I'll do a PR for that later.

Reply:
Looks good, but the game object needs to be held in State, then we can derive players, currentRound etc from that state.

Reply:
Yep - do we need to do that now though or when we get to it?

Reply:
added